.storybook/assets/base.css

```diff
@@ -29,16 +29,31 @@ body {
 }
 
 .spectrum {
+	/* Gradient that changes with the color theme. */
+	--spectrum-examples-gradient: linear-gradient(45deg, var(--spectrum-magenta-1500), var(--spectrum-blue-1500));
+
+	/* Gradients that do not change with the color theme, for use in static color backgrounds. */
+	--spectrum-examples-gradient-static-black: linear-gradient(45deg, rgb(255 241 246), rgb(238 245 255));
+	--spectrum-examples-gradient-static-white: linear-gradient(45deg, rgb(64 0 22), rgb(14 24 67));
+
 	color: var(--spectrum-neutral-content-color-default);
 	background-color: var(--spectrum-background-base-color);
-	-webkit-tap-highlight-color: rgba(0, 0, 0, 0%);
+	-webkit-tap-highlight-color: rgb(0, 0, 0, 0%);
```

Comment:
This was just an auto-fix ala VSCode stylelinting

Reply:
Can you say more about this and the other change below from hsla to hsl? I would have thought it'd be more correct to use rgba and hsla, but it seems like that's not the case.

Reply:
I just learned about this the other day too! Apparently rgba and hsla are actually just aliases for rgb and hsl! PostCSS preset env tooling suggests we use the color function directly instead of the alias.